 * Running on http://legend02.pc.cc.cmu.edu:53026/ (Press CTRL+C to quit)
 * Running on http://legend02.pc.cc.cmu.edu:53026/ (Press CTRL+C to quit)
 * Running on http://legend02.pc.cc.cmu.edu:53026/ (Press CTRL+C to quit)
172.16.1.1 - - [17/Apr/2016 20:14:51] "POST /tensor/run HTTP/1.1" 200 -
I tensorflow/core/common_runtime/local_device.cc:40] Local device intra op parallelism threads: 8
I tensorflow/core/common_runtime/direct_session.cc:58] Direct session inter op parallelism threads: 8
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 811, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 764, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/bicadmin/workspace-dt/core-code-base/flask/run_demo.py", line 158, in run_mlp_train
    run_mlp(lr=lr, hidden_weights=h_weights, num_iter=it, train_dir=train, output_dim=o_dim, mode="train", saved_model_path=model_path, saved_weights_path=weights_path, output_file=output)
  File "/home/bicadmin/workspace-dt/core-code-base/flask/run_demo.py", line 146, in run_mlp
    weights[:,i] = weights[:,i].tolist()
TypeError: list indices must be integers, not tuple

Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 811, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 764, in run
    self.__target(*self.__args, **self.__kwargs)
  File "hello.py", line 84, in run_exp
    datagen, headers = multipart_encode({"token":"lafyyjveotnehialteikeniotjim", "username":user_name, "name":name, "model": open(model_output, "rb"), "train_out": open(train_output, "rb"), "test_out": open(test_output, "rb"), "weights" : open(weights_output, "rb"), "status":"Completed"})
IOError: [Errno 2] No such file or directory: u'files/jw_abcde_weights.pckl'

 * Running on http://legend02.pc.cc.cmu.edu:53026/ (Press CTRL+C to quit)
172.16.1.1 - - [17/Apr/2016 20:32:54] "POST /tensor/run HTTP/1.1" 200 -
I tensorflow/core/common_runtime/local_device.cc:40] Local device intra op parallelism threads: 8
I tensorflow/core/common_runtime/direct_session.cc:58] Direct session inter op parallelism threads: 8
 * Running on http://legend02.pc.cc.cmu.edu:53026/ (Press CTRL+C to quit)
172.16.1.1 - - [18/Apr/2016 10:22:28] "POST /tensor/run HTTP/1.1" 200 -
I tensorflow/core/common_runtime/local_device.cc:40] Local device intra op parallelism threads: 8
I tensorflow/core/common_runtime/direct_session.cc:58] Direct session inter op parallelism threads: 8
172.16.1.1 - - [18/Apr/2016 10:41:05] "POST /tensor/run HTTP/1.1" 200 -
172.16.1.1 - - [18/Apr/2016 11:22:12] "POST /tensor/run HTTP/1.1" 200 -
